daemon/logger/splunk/splunk.go

```diff
@@ -0,0 +1,256 @@
+// +build linux
```

Comment:
Ok, so per @glennblock we do not support Windows right now. I mean it should work, but we have not tested it.
Should I keep this line for now?
Btw, do I need to have special file "splunk_unsupported.go"?